include.js

```diff
@@ -5890,6 +5890,13 @@ let cards = [
     author: 'Graphitexhd',
     githubLink: 'https://github.com/graphitexhd'
   },
+  {
```

Comment:
Hello @MuhannadBaraghith  Please add your contribution to the end of the cards list. Thank You for the great work!